### PR TITLE
Blog post about turning off VPNs and trackers

### PR DESCRIPTION
**Pull request by andyjih**

### CODE REVIEW

File: pages/blog/dont-forget-your-vpn.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Don’t Forget Your VPNs and Tracker Blockers"
+date: "2021-01-27"
+author: "Andy Jih"
+pullQuote: "Ad blockers are great, until they’re not"
+image: "dont-forget-your-vpn/intercom_blocked.png"
+tags: [product, notes]
+---
+
+I was testing the new [Intercom](https://intercom.com) plugin that our team had built, and ran into a strange snag. When adding apps to Grouparoo, we have this handy “Test Connection” button that checks to make sure the connection details and credentials work. When I was testing this new Intercom plugin, something strange appeared this time around.
```
Comment (evantahler) [inline pages/blog/dont-forget-your-vpn.mdx:10]: Is there anywhere on our site we talk about intercom to link to?

File: pages/blog/dont-forget-your-vpn.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+<Image
+  centered
+  alt="Steps of a simple sync"
+  src="dont-forget-your-vpn/intercom_blocked.png"
+  width={845}
+  height={586}
+/>
+
+I was testing locally, but the Test Connection button actually talks to an external API, so I was confused that the connection failed due to something from localhost.
```
Comment (evantahler) [inline pages/blog/dont-forget-your-vpn.mdx:20]: Add some text that described what is shown in the image, "The connection didn't work!" (for vision-impared people, RSS readers, etc)

File: pages/blog/dont-forget-your-vpn.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+<Image
+  centered
+  alt="Steps of a simple sync"
+  src="dont-forget-your-vpn/intercom_unblocked.png"
+  width={842}
+  height={629}
+/>
+
+This experience served as yet another reminder that as tools become more sophisticated at blocking ads and trackers like [Google Analytics](https://analytics.google.com), Facebook Ads, and [Segment](https://segment.com), front-end events will become less useful to businesses (and less creepy to the general public!).
```
Comment (evantahler) [inline pages/blog/dont-forget-your-vpn.mdx:44]: Tie it back to Grouparoo maybe?  
> As end-users become more sophisticated and concerned about their online privacy, the effectivness of event tracking tools like Segment and Intercom will be reduced.  That's why Grouparoo uses your source-of-truth data whenever possible.

File: pages/blog/dont-forget-your-vpn.mdx (hunk 1 above)
Comment (andyjih) [reply pages/blog/dont-forget-your-vpn.mdx:10]: Not yet-- I'll update the link when that page is up!

File: pages/blog/dont-forget-your-vpn.mdx (hunk 2 above)
Comment (andyjih) [reply pages/blog/dont-forget-your-vpn.mdx:20]: Oops, forgot to update the Alt texts

File: pages/blog/dont-forget-your-vpn.mdx (hunk 3 above)
Comment (andyjih) [reply pages/blog/dont-forget-your-vpn.mdx:44]: Wasn't sure how much to make it a selling point. I'll try something lightweight

**Review by evantahler — APPROVED** [review]

**Review by seancdavis — APPROVED** [review]
Love it!

**Pull request merged**